habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -279,21 +281,21 @@ def reconfigure(self, config: "DictConfig", ep_info: RearrangeEpisode):
 
         if self.first_setup:
             self.first_setup = False
-            self.robots_mgr.first_setup()
+            self.agents_mgr.first_setup()
             # Capture the starting art states
             self._start_art_states = {
                 ao: ao.joint_positions for ao in self.art_objs
             }
 
-    def get_robot_data(self, agent_idx: Optional[int]):
+    def get_agent_data(self, agent_idx: Optional[int]):
         if agent_idx is None:
-            return self.robots_mgr[0]
+            return self.agents_mgr[0]
         else:
-            return self.robots_mgr[agent_idx]
+            return self.agents_mgr[agent_idx]
 
     @property
     def num_robots(self):
-        return len(self.robots_mgr)
+        return len(self.agents_mgr)
 
     def set_robot_base_to_random_point(
```

Comment:
Can we change everything in `rearrange_sim.py` to no longer reference `robot` but instead reference `articulated_agent`? We should be consistent with the naming throughout this file. It probably makes sense to use the `robot` naming in sensors that are for robot specific tasks, but `rearrange_sim` is general for everything, so it should use the general language.